test/e2e/basic/csvs.go

```diff
@@ -30,35 +30,43 @@ const (
 	performanceOperatorDeploymentName = "performance-operator"
 )
 
-var _ = Describe("[basic][clusterserviceversion] ClusterServiceVersion listing", Ordered, func() {
+var _ = Describe("[basic][clusterserviceversion] ClusterServiceVersion listing", func() {
 	var cli client.Client
 	var paginationLimit uint64
 	var namespace corev1.Namespace
 	var baseOptions []client.ListOption
-	BeforeAll(func() {
-		var err error
+	var count int
 
-		By("Getting new client")
-		cli, err = newClient()
-		Expect(err).To(Succeed())
+	BeforeEach(func() {
+		if count == 0 {
```

Comment:
this works iff the test are strictly serial, are them guaranteed serial? otherwise it seems racy (or wrong)

Reply:
your are totally right.
Thanks for the catch. 

I can't find a way to run the test sequentially using ginkgo v1, so I am thinking about creating the `client` in `BeforeSuite` and let the namespace be created/destroyed on `BeforeEach/AfterEach`.
I am gonna "merge" the `It` clauses so the code is only executed once to save execution time.